apps/admin-gui/src/app/shared/components/hosts-list/hosts-list.component.html

```diff
@@ -0,0 +1,59 @@
+<div class="card mt-3" [class.hide-table]="exporting" [hidden]="dataSource.filteredData.length === 0 || hosts.length === 0">
+  <div class="card-body">
+    <perun-web-apps-table-options (exportEnd)="exporting = false" (exportStart)="exporting = true" [exporter]="exporter" class="ml-auto"></perun-web-apps-table-options>
```

Comment:
class="ml-auto" - does not have any use, if you remove it the button stays on the same place